src/cli/README_SRP.md

```diff
@@ -100,10 +92,27 @@ Done
 Done
 > srp client service add my-service _ipps._tcp 12345
 Done
+> srp client autostart enable
+Done
+```
+
+The last command enables the auto-start mode on the client which then monitors the network data to discover available SRP servers within the Thread network and automatically starts the client.
+
+Alternatively, the client can be started manually using the `srp client start`.
+
+```bash
+> netdata show
+Prefixes:
+Routes:
+Services:
+44970 5d c002 s 8400
+Done
```

Comment:
Move this and the network data description line below above the "Alternatively, ..." line?